### PR TITLE
DCS-812: login integration test added. 

### PR DESCRIPTION
**Pull request by GurnankCheema**
Test needed to verify the application logs current user out if a new user logs in.

### CODE REVIEW

File: integration-tests/plugins/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = on => {
 
     getLoginUrl: auth.getLoginUrl,
 
-    stubLogin: () => Promise.all([auth.stubLogin({}), elite2api.stubUser(), elite2api.stubUserCaseloads()]),
+    stubLogin: user => Promise.all([auth.stubLogin(user), elite2api.stubUser(user), elite2api.stubUserCaseloads()]),
```
Comment (andrewrlee) [inline integration-tests/plugins/index.js:16]: could you default this to `{}` here and avoid having to pass in empty object everywhere?

File: integration-tests/plugins/index.js [hunk 2]
```diff
@@ -40,5 +40,7 @@ module.exports = on => {
     stubFindUsers: auth.stubFindUsers,
 
     stubUnverifiedUserDetailsRetrieval: auth.stubUnverifiedUserDetailsRetrieval,
+
+    stubVerifyToken: (active = true) => auth.stubVerifyToken(active),
```
Comment (andrewrlee) [inline integration-tests/plugins/index.js:44]: `stubVerifyToken: auth.stubVerifyToken,` ? so you don't need to do the defaulting in two places?

File: integration-tests/plugins/index.js (hunk 1 above)
Comment (GurnankCheema) [reply integration-tests/plugins/index.js:16]: yes

File: integration-tests/plugins/index.js (hunk 2 above)
Comment (GurnankCheema) [reply integration-tests/plugins/index.js:44]: correct

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**